packages/manager/apps/pci-volume-snapshot/src/api/api.types.ts

```diff
@@ -0,0 +1,55 @@
+export type TSnapshot = {
+  id: string;
+  creationDate: string;
+  name: string;
+  description: string;
+  size: number;
+  volumeId: string;
+  region: string;
+  status: 'available' | 'creating' | 'deleting' | 'error' | 'error_deleting';
```

Comment:
We could be more exhaustive and create dedicated types for `enums` or `Records`, so you can easily import and manipulate them without knowing the whole structure.